doc/developer.md

```diff
@@ -22,21 +22,24 @@ We recommend this for development work ONLY.
     pip install setuptools --upgrade
     ```
     
-3. Install requirements.txt
-    ```
-    pip install -r requirements.txt
-    ```
-
-4. Clone the repo to `/var/www/html`
+3. Clone the repo to `/var/www/html
+    ```
+    git clone https://github.com/apel/rest.git /var/www/html
+    ```
+
+4. Install requirements.txt
     ```
-    git clone https://github.com/apel/rest.git /var/www/html
+    pip install -r /var/www/html/requirements.txt
     ```
 
 5. Create the database
     ```
-    mysql -u root -e "create database apel_rest"
-    mysql -u root apel_rest < schemas/cloud.sql
-    mysql -u root apel_rest < schemas/cloud-extra.sql
+    mysql -u root -e "create user 'apel'@'localhost';"
+    mysql -u root -e "GRANT ALL PRIVILEGES ON apel_rest.* TO 'apel'@'localhost';"
+    mysql -u root -e "FLUSH PRIVILEGES;"
+    mysql -u apel -e "create database apel_rest"
+    mysql -u apel apel_rest < /var/www/html/schemas/10-cloud.sql
```

Comment:
The numbers are for that odd Docker loading order thing, right?

Reply:
yes